config/settings/base.py

```diff
@@ -133,12 +133,25 @@
 
 # GLOBAL VARIABLES
 
-# Apparent term numbering system (unconfirmed):
-#   Fall: 1{last 2 digits of ending year}2
-#   Spring: 1{last 2 digits of ending year}4
+# ACTIVE_TERMS defines the terms that are scraped by the scraper
+# Term numbering system:
+#   First digit: always a 1 -- not clear why
+#   Second and Third digits: last 2 digits of ending year of schoolyear
+#   Fourth digit: the season number, as follows
```

Comment:
Mention that 2 and 4 is most often used since we have fall/spring semesters. We don't wanna make outsiders think we're on a quarter system or something